pkg/prommetrics/adapter/adapter_prommetrics.go

```diff
@@ -45,6 +45,15 @@ var (
 		},
 		metricLabels,
 	)
+	scalerMetricsLatency = prometheus.NewGaugeVec(
```

Comment:
Metrics exposed in Adapter are deprecated, so we shouldn't add a new metrics here. Could you please remove this?

Reply:
Sure, removed